opennms-doc/guide-admin/src/asciidoc/text/provisioning/getting-started.adoc

```diff
@@ -13,9 +13,12 @@ We'll start with _Auto Discovery_, but first, we should quickly review the confi
 
 ==== Provisioning the SNMP Configuration
 
-{opennms-product-name} requires that the SNMP configuration to be properly setup for your network in order to properly understand Network and Node topology as well as to automatically enabled performance data collection.
+{opennms-product-name} requires SNMP configuration to be properly setup for your network in order to properly understand Network and Node topology as well as to automatically enable performance data collection.
 Network topology is updated as nodes (a.k.a. devices or hosts) are provisioned.
-Navigate to the _Admin/Configure SNMP Community Names_ as shown below.
+Navigate to the _Admin/Configure SNMP Community Names by IP address_ as shown below.
+
+.Configuring SNMP community names
+image:../images/provisioning/00029.png[]
```

Comment:
Hi Ben,

That's strange since it's the way how image macros have always been used in OpenNMS' doc.

Look at file https://github.com/StCyr/opennms/blob/8f9af12e51ca423945bb5785cfdb23edc486ccdb/opennms-doc/guide-admin/src/asciidoc/text/provisioningexamples.adoc for example

It also dosen't render properly on github while it renders good here: 

http://docs.opennms.org/opennms/releases/18.0.2/guide-admin/guide-admin.html#_provisioning_examples

I'm afraid fixing the issue for github's rendering would break the HTML and PDF rendering builds.
Cyrille